### PR TITLE
adobe-air-sdk moved in homebrew/binary

### PR DESCRIPTION
**Pull request by bfontaine**
`brew audit --strict` complains about system OpenSSL linking but we can’t do anything since we’re installing the binaries, not compiling from source code (adding `depends_on "openssl"` as suggested by the warning doesn’t change anything).

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Is the source provided anywhere for this?

**Comment by bfontaine** [comment]
I don’t think so.

File: Library/Formula/adobe-air-sdk.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ class AdobeAirSdk < Formula
     sha1 "3feaadf534f5f186286eb33cb4461d7e78cdb91d"
   end
 
+  depends_on "openssl"
```
Comment (mikemcquaid) [inline Library/Formula/adobe-air-sdk.rb]: Just remove this if it does nothing :sob:

**Comment by mikemcquaid** [comment]
@Homebrew/owners thoughts on migrating this to homebrew-binary?

File: Library/Formula/adobe-air-sdk.rb (hunk 1 above)
Comment (bfontaine) [inline Library/Formula/adobe-air-sdk.rb]: Done.

**Comment by jacknagel** [comment]
let's move it

**Comment by mikemcquaid** [comment]
@bfontaine Would you mind creating the relevant homebrew-binary PR and migration and removal in this PR? Thanks!

**Comment by bfontaine** [comment]
Yep I’ll do that.

**Comment by bfontaine** [comment]
@mikemcquaid: Done! See also Homebrew/homebrew-binary#191.

**Pull request closed without merging**